src/referencePackages/src/system.formats.asn1/6.0.0/System.Formats.Asn1.6.0.0.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
System.Formats.Asn1.6.0.0 contains a vulnerability, so we can't add it to SBRP (see https://github.com/dotnet/source-build/issues/4518). If there's a reference to this package, you can manually change the reference to 6.0.1 and add a comment above it saying that it was changed manually (see https://github.com/dotnet/source-build-reference-packages/pull/992/files)

Reply:
thanks, I removed it